tutorials/hana-clients-install/hana-clients-install.md

```diff
@@ -80,7 +80,7 @@ The SAP HANA client can be used to connect to different versions of SAP HANA.  F
 
     ![Version of HDBSQL](command-Prompt.png)
 
-    > To uninstall at a later time, run `hdbuninst` from the folder where the client is installed.  For example ~/sap/hdbclient/install/hdbuninst.  
+    > To uninstall at a later time, run `hdbuninst` from the folder where the client is installed.  For example `c:\sap\hdbclient\install\hdbuninst`.  
```

Comment:
remove one space before "For example"

Reply:
It seems that HTML ignores the additional space. So there is no need to remove it.